cli/cmdcontext/cmdcontext.go

```diff
@@ -61,6 +61,48 @@ func (tntCli *TarantoolCli) GetVersion() (version.Version, error) {
 	return tntCli.version, nil
 }
 
+// TtCli describes tt executable.
+type TtCli struct {
+	// Executable is a path to Tt executable.
+	Executable string
+	// Tt version.
+	version version.Version
+}
+
+// GetVersion returns and caches the tt version.
+func (ttCli *TtCli) GetVersion() (version.Version, error) {
+	if ttCli.version.Str != "" {
+		return ttCli.version, nil
+	}
+
+	if ttCli.Executable == "" {
+		return ttCli.version, fmt.Errorf(
+			"tt executable is not set, unable to get tarantool version")
+	}
+	output, err := exec.Command(ttCli.Executable, "version").Output()
```

Comment:
This will not work, because `tt` without parameters will search for tt.yaml up from current dir. So, this executable may run another one, and we will get the version from other `tt`, not `ttCli.Executable`.